src/Views/Textile/Step.elm

```diff
@@ -130,36 +127,37 @@ airTransportRatioField { current, updateAirTransportRatio } =
 
 
 dyeingMediumField : Config msg modal -> Html msg
-dyeingMediumField { inputs, updateDyeingMedium } =
-    div [ class "d-flex justify-content-between align-items-center fs-7" ]
-        [ label [ class "text-truncate w-25", for "dyeing-medium", title "Teinture sur" ]
-            [ text "Teinture sur" ]
-        , [ DyeingMedium.Yarn, DyeingMedium.Fabric, DyeingMedium.Article ]
-            |> List.map
-                (\medium ->
-                    option
-                        [ value <| DyeingMedium.toString medium
-                        , selected <| inputs.dyeingMedium == Just medium || inputs.product.dyeing.defaultMedium == medium
-                        ]
-                        [ text <| DyeingMedium.toLabel medium ]
-                )
-            |> select
-                [ id "dyeing-medium"
-                , class "form-select form-select-sm w-75"
-                , onInput
-                    (DyeingMedium.fromString
-                        >> Result.withDefault inputs.product.dyeing.defaultMedium
-                        >> updateDyeingMedium
+dyeingMediumField { inputs, updateDyeingMedium, showAdvancedFields } =
+    showIf showAdvancedFields <|
```

Comment:
Would be better to move up the `showIf (...) <|` outside each view function and gather them in the calling view, to keep the upper layout at the same place

Reply:
done in https://github.com/MTES-MCT/ecobalyse/pull/607/commits/298cb78c55fb254cad72a2407211f1917fe4e5e5